src/exercises/debug-and-disassemble/README.md

```diff
@@ -5,7 +5,9 @@ RISC-V and CHERI-RISC-V programs. It draws attention to differences in
 program structure and code generation, particularly relating to control
 flow, between the two compilation targets.
 
-First, use `llvm-objdump` to disassemble and explore the two binaries
+First, use `llvm-objdump` on the host (which you can find as part of the
+CHERI SDK, at `~/cheri/output/sdk/bin/llvm-objdump`, if you have not changed
+the cheribuild default paths) to disassemble and explore the two binaries
```

Comment:
```suggestion
First, use `llvm-objdump` on the host (which you can find at `~/cheri/output/sdk/bin/llvm-objdump`, unless you have altered `cheribuild`'s default paths) to disassemble and explore the two binaries
```
(wrapped appropriately)
The CHERI SDK is a bit redundant given the path, and that exact wording about default paths is what we use in the introduction when pointing at Clang